src/bitmessageqt/__init__.py

```diff
@@ -1890,11 +1902,13 @@ def addRow (address, label, type):
             newRows[address] = [label, AccountMixin.NORMAL]
 
         completerList = []
-        for address in sorted(oldRows, key = lambda x: oldRows[x][2], reverse = True):
-            if address in newRows:
-                completerList.append(unicode(newRows[address][0], encoding="UTF-8") + " <" + address + ">")
-                newRows.pop(address)
-            else:
+        for address in sorted(
+            oldRows, key=lambda x: oldRows[x][2], reverse=True
+        ):
+            try:
+                completerList.append(
```

Comment:
Have you tested it with UTF-8 characters (e.g. cyrillic will probably be the easiest for you to test)?

Reply:
retested just now